src/system/game-data.ts

```diff
@@ -1791,6 +1791,35 @@ export class GameData {
     });
   }
 
+  /**
+   * @param {PokemonSpecies} species
+   * @returns {boolean} whether root species has been unlocked for given {@linkcode PokemonSpecies}
+   */
+  isRootSpeciesUnlocked(species: PokemonSpecies): boolean {
+    return !!this.dexData[species.getRootSpeciesId()]?.caughtAttr;
+  }
+
+  /**
+   * Unlocks nature for a given {@linkcode PokemonSpecies} and its prevolutions.
+   * Will fail silently if root species has not been unlocked
+   * @param {PokemonSpecies} species
+   * @param {Nature} nature
```

Comment:
```suggestion
   * Unlocks the given {@linkcode Nature] for a {@linkcode PokemonSpecies} and its prevolutions.
   * Will fail silently if root species has not been unlocked
```
No need to add the @ params if they are self explanatory